### PR TITLE
ext: stm32cube: stm32f4xx: shift I2SR field in PLLI2SCFGR register

### PR DESCRIPTION
**Pull request by avisconti**
This PR is fixing two bugs into ST CubeMX code for stm32f4xx and stm32f7xx. Both the two bugs are related to the routines to configure the PLLI2S where the PLLR field was not properly shifted.

ST BUG Tracker IDs:  50108 and 50086

### CODE REVIEW

**Comment by avisconti** [comment]
@erwango 
Please note that for the moment in README file I put:
    ST Bug tracker ID: TO BE ASSIGNED

It would be nice if we could get a number before the end of the week.

File: ext/hal/st/stm32cube/stm32f4xx/README [hunk 1]
```diff
@@ -43,3 +43,10 @@ Patch List:
     Impacted files:
      drivers/include/stm32f4xx_hal_conf.h
     ST Bug tracker ID: NA. Not a stm32cube issue
+
+   *Add correct shifting to I2SR field in PLLI2SCFGR register
+     The I2SR field should be shifted by RCC_PLLI2SCFGR_PLLI2SR_Pos when the PLLI2SCFGR register
+     is read or written.
+    Impacted files:
+     drivers/include/stm32f4xx_ll_rcc.h
+    ST Bug tracker ID: TO BE ASSIGNED
```
Comment (erwango) [inline ext/hal/st/stm32cube/stm32f4xx/README]: 50086

**Review by erwango — COMMENTED** [review]
Provided internal ST tracker.
Do you know if this bug is also present for other series?

File: ext/hal/st/stm32cube/stm32f4xx/README (hunk 1 above)
Comment (avisconti) [reply ext/hal/st/stm32cube/stm32f4xx/README]: >> Provided internal ST tracker.

ok, good.

>> Do you know if this bug is also present for other series

Well, no, I don't know.
I can check it quickly tomorrow. Let's say that the PLLI2S is not present in all the series if I got it properly. Maybe only stm32f4xx and stm32f10x. But I need to check carefully.

**Comment by avisconti** [comment]
@erwango 
I think that also stm32f7xx has same bug

**Comment by erwango** [comment]
@avisconti , could you create the same patch in this PR for F7 series as well?
Bug tracker : 50108

Thanks!

**Review by erwango — APPROVED** [review]

**Pull request merged**